ocaml/testsuite/tests/warnings/w53_across_cmi.ml

```diff
@@ -0,0 +1,18 @@
+(* TEST
+   readonly_files = "w53_without_cmi.ml w53_with_cmi.mli w53_with_cmi.ml";
+   setup-ocamlc.byte-build-env;
+   all_modules = "w53_without_cmi.ml w53_with_cmi.mli w53_with_cmi.ml";
+   flags = "-w +A-70";
+   compile_only = "true";
+   ocamlc.byte;
+   all_modules = "w53_across_cmi.ml";
+   flags = "-alert +all -w +A-33-70";
+   ocamlc.byte;
+   check-ocamlc.byte-output;
+*)
```

Comment:
I believe the reference file for this test is missing.

Reply:
Indeed, fixed. And we will find out whether the output agrees when Mark gets the testsuite running.